docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,611 @@
+}
+```
+
+Also under the `client` object add another property called `clientSignedCert` and under this add a property called `path` that has a string variable leading to the identities signed certificate. Again note that the provided path is workspace relative. In this example it is located at **fabric-samples** -> **test-network** -> **organizations** -> **peerOrganizations** -> **org1.example.com** -> **users** -> **Admin@org1.example.com** -> **msp** -> **signedcerts** -> **admin@org1.example.com-cert.pem**
+
+This is what should be added to the `client` object:
+
+```json
+"clientSignedCert": {
+    "path": "../fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/signcerts/Admin@org1.example.com-cert.pem"
+}
+```
+
+The `clients` object should now look like this: 
+
+```json
+    "clients": {
+        "Admin@org1.example.com": {
+            "client": {
+                "organization": "Org1",
+                "connection": {
+                    "timeout": {
+                        "peer": {
+                            "endorser": "300"
+                        }
+                    }
+                },
+                "credentialStore": {
+                    "path": "tmp/hfc-kvs/org1",
+                    "cryptoStore": {
+                        "path": "tmp/hfc-kvs/org1"
+                    }
+                },
+                "clientPrivateKey": {
+                    "path": "../fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/keystore/priv_sk"
+                },
+                "clientSignedCert": {
+                    "path": "../fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/signcerts/Admin@org1.example.com-cert.pem"
+                }
+            }
+        }
+    }
+```
+
+#### Channels Object
+The Fabric connector for Caliper requires assistance when creating connections to a Fabric network. A `channels` object must be provided that lists the smart contracts that may be interacted with.
+
+Add the known channel, `mychannel`, that was created by the `test-network` into the `channels` section of the Caliper network configuration file and give it the property `created` with a boolean value of true. We must list the smart contracts available on this channel as objects within an array named `contracts`. Each smart contract object has two properties, `id` and `version`. `id` specifies contract ID; in this case it is `asset-transfer-basic`. The version is the specific contract version; in this case it is `1.0.0`. Add this as an object within the array such that the resulting `channels` object within the Caliper network configuration file becomes:
+
+```json
+    "channels": {
+        "mychannel": {
+            "created" : true,
+            "contracts": [
+                {
+                    "id":"asset-transfer-basic",
```

Comment:
For me when using node the id was just `basic`. Would this be different for other chaincode languages like `Go` ?

Reply:
For the `v2.1.1` samples repo and the JS CC, I see the `fabcar` name in the script invocations :game_die: 
Btw, does the CC version have any significance beside CC installation in the old-style CC deploy process?